### DISCUSSION TITLE
cn/2024/12/04/info-scarcity/

### DISCUSSION THREAD

**Discussion by giscus** [hongtaoh/hongtaoh.github.io#191] [Announcements]
# cn/2024/12/04/info-scarcity/

https://hongtaoh.com/cn/2024/12/04/info-scarcity/

**Comment by Liam-Zhong** [comment]
感觉博主有点偏激了，这个世界上不是所有的信息都要做有用之用。
>韩国首尔江南区目前有多少人在睡觉，请告>诉我。
>达拉斯飞往上海的 AA 127 航班，目前的时>速是多少？
>现在有多少人在同时听周杰伦的歌？
>全世界在这一秒钟有多少盏灯在开着？
>北京中关村 59 号大门附近的温度和湿度分>别是多少？
>全世界有多少异性和我的性格匹配？她们所>在位置的经纬度请列出来。
上述很多东西可以基于概率学计算出来，记得本世纪还不乏这样的美谈（比如费米）。当然，这样分析的确算不上信息爆炸，不过针对像周杰伦和异性这种话题，前者统计下几个头部听歌软件数据即可，后者在几年前则是不敢想的事情。虽然我也觉得网上没啥值得注意的信息，但与其说不是信息爆炸，倒不如说信息熵太低。至于你设想的社会形态……我感觉有生之年是看不到了，毕竟地球上还有很多但都吃不饱的国家。

**Reply by Liam-Zhong** [reply]
嗨，搞错格式了

**Reply by hongtaoh** [reply]
感谢分享你的看法。听一听不同的声音挺好的。

我也觉得我想的在我们有生之年比较难看到，但不妨碍想象。

**Comment by weifanz** [comment]
感觉隐私让渡这个问题，还涉及到让渡多少，让渡给谁，管理权究竟在谁手上，或者数据能否进行打乱。如果商业公司与机构做不到透明化，或者如果我不知道数据传输中间环节发生了什么，或者我对一项产品并没有需求的时候，我是不信任把自己的隐私在没打乱的情况下交给商业公司的。或者从另一个方面来讲，有的时候，灰色地带是人与人保持信任/互不干涉的必要条件（可以保持信任也可以摧毁信任），否则如果真出现了《真相时刻》（某节目）那种场景，这有点恐怖。当然，无关紧要、宏观的数据是个例外（例如车流量、交易量），但是微观的数据让渡多少，这是个争议。或者，“无关紧要”及“不能碰”的数据，它们如何定义，究竟是通过加密还是干脆不收集的方式保护“灰色地带”，也是个问题。

**Reply by hongtaoh** [reply]
很有深度的评论。最后数据是掌握在政府手里还是科技企业手里，真说不好。我的预测是掌握在人工智能手里。

**Comment by botsp** [comment]
眼前一亮的观点，对人来说可能已经信息过载了; 对于未来机器，他们的"大数据时代"可能刚刚开始
